### DISCUSSION TITLE
Five things you didn't know you could do on GitHub in your day job

### DISCUSSION THREAD

**Discussion by tuves** [community/community#50827] [Announcements]
Delivering software involves more than just throwing down some lines of code. To truly be agile and deliver high-quality software, we need to think about our tooling to boost our productivity.

So how do you best manage these? How can GitHub help in your day job? That’s what we’re going to explore in this series - GitHub in My Day Job. Every week, Senior Enterprise Advocates April Edwards and Christopher Harrison will welcome guests to demonstrate tooling, explore best practices, and highlight how GitHub can streamline your development process.

In our first episode, join us as April and Christopher show you a few tips and tricks that are built into GitHub. [Tune in](https://www.linkedin.com/events/7044615491008561152/about/) to learn how to accelerate your day to day tasks. Bring your questions, and we’ll help you become more productive and make your day job just a little easier.

![LI](https://user-images.githubusercontent.com/49915749/227231516-dcea6b1d-e270-4d53-bb33-a7511ea8e750.jpeg)

**Click [here](https://www.linkedin.com/events/7044615491008561152/about/) to register to the event** 🚀

**Comment by ashleychan28** [comment]
😅😅🤫🫢

**Reply by m-housni** [reply]
I'm in. Thank you

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
